include/cutlass/epilogue/thread/activation.h

```diff
@@ -193,7 +192,20 @@ struct HardSwish {
     minimum<T> mn;
     maximum<T> mx;
     T relu6 = mn(mx(x + T(3), T(0)), T(6));
-    return x * (relu6 / T(6));
+    return x * relu6 / T(6);
```

Comment:
return x * relu6 * T(0.16666667);

Reply:
Base class still needs to use `/ 6`.  Not all data types supports `* 0.167`.